### ISSUE TITLE
Nerdminer Stops Mining

### ISSUE THREAD

**Issue by TrustinDo** [BitMaker-hub/NerdMiner_v2#99]
After installing the new software my miners would be at 45k/s and then 0 and they back to 45k/s and would repeat that ever few minutes.
![Uploading 802EE4A8-FF29-4AC9-980D-2A320DFF40E3.jpeg…]()

**Comment by lupinglui76** [comment]
i got the same problem...

**Comment by Bergge** [comment]
I had the same problem where the hashrate fluctuated between 45 KH/s and 0 KH/s. I found read [Issue#78](https://github.com/BitMaker-hub/NerdMiner_v2/issues/78) where a similar problem was discussed and the solution worked for me. I powered my nerdminer through the USB port of my home NAS. After using a "real" charger everything worked fine. So maybe the solution ist to connect your nerdminer to a normal charger and don't use the USB port of your NAS or your router.
